.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+python:
+  - "2.7"
+  - "3.3"
+  - "3.4"
```

Comment:
[`3.3` is in security-fix only mode](https://www.python.org/dev/peps/pep-0398/#id5).